LibTest/ffi/UnsignedLong/UnsignedLong_A01_t01.dart

```diff
@@ -33,7 +33,8 @@ void main() {
     p1.value = 0x80000000;
     Expect.equals(2147483648, p1.value);
     p1.value = -42;
-    Expect.equals(4294967254, p1.value);
+    Expect.isTrue(4294967254 == p1.value || // 32-bit representation
```

Comment:
same

Reply:
If the same number is returned in case of 64-bit, then the comments are correct. `-42` is transformed to `4294967254` in case of 32 bit and remains `-42` in case of 64 bit.

Reply:
(I'd love to hear the answer, too, @dcharkes! ;-)

Reply:
same